i18n/en.pot

```diff
@@ -5,143 +5,135 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
-"POT-Creation-Date: 2019-06-04T14:36:02.608Z\n"
-"PO-Revision-Date: 2019-06-04T14:36:02.608Z\n"
+"POT-Creation-Date: 2019-06-13T06:24:14.176Z\n"
+"PO-Revision-Date: 2019-06-13T06:24:14.176Z\n"
 
-msgid "Instance Configurator"
-msgstr ""
-
-msgid "Organisation Units Sync"
+msgid "<No value>"
 msgstr ""
 
-msgid "Data Elements Sync"
+msgid "Server name (*)"
```

Comment:
[Note: not for this PR] Maybe have translations without the `(*)` required field suffix and add them programmatically whenever needed.